spec/requests/posts_controller_spec.rb

```diff
@@ -1,27 +0,0 @@
-require 'rails_helper'
-
-RSpec.describe 'Posts Controller', type: :request do
-  describe 'GET #index' do
```

Comment:
- [ ] You have failing tests, kindly fix them.
![image](https://user-images.githubusercontent.com/81932478/192390259-249cffec-b7ac-4480-837d-345991b03776.png)

![image](https://user-images.githubusercontent.com/81932478/192390291-80641c9a-45cb-4de1-b8fa-07ea045189a4.png)

![image](https://user-images.githubusercontent.com/81932478/192390492-c6fe4bf1-b14e-4d8d-8d9a-23691293c5df.png)

Reply:
Hey @Munsa1 , this folder has been removed, I think you pulled the wrong branch